app/assets/stylesheets/admin/questions/_questions.scss

```diff
@@ -78,3 +78,12 @@ section.questionnaires li.jqtree_common .jqtree-element
 .linked-document .help-block {
   margin: .25em 0;
 }
+
+.answer-wrapper.tree-view .input-with-postfix.form-element {
```

Comment:
Expecting to reuse the 'input with postfix' and 'input-postfix' classnames here for currency. Will need to rename to include prefix.

Reply:
Also, I'm not loving depending on simple_form styles here, but these styles get overriden if I only using custom classes